### PR TITLE
Add rsyslog service start function

### PR DESCRIPTION
**Pull request by kylazhang**
Signed-off-by: Kylazhang <weizhan@redhat.com>

### CODE REVIEW

File: virttest/utils_misc.py [hunk 1]
```diff
@@ -4559,3 +4560,19 @@ def get_pid(name, session=None):
...
+    Start rsyslogd service
+    """
+    try:
+        utils_path.find_command("rsyslogd")
+    except utils_path.CmdNotFoundError:
+        exceptions.TestError("No rsyslogd command found.")
+    rsyslogd = service.Factory.create_service('rsyslog')
+    if not rsyslogd.status():
+        logging.info("Need to start rsyslog service")
+        if not rsyslogd.start():
```
[the remark this answers is not in the document]
Comment (kylazhang) [reply virttest/utils_misc.py]: ok, glad to see you again :) will update soon. Please help to approve it asap, thanks

[the remark this answers is not in the document]
Comment (kylazhang) [reply virttest/utils_misc.py]: Seems I have make some mistakes, the service may return process.run result, so I need to check the exit_status to make sure it succeed or fail

**Comment by kylazhang** [comment]
@xutian @chunfuwen please help to review and merge as related tp-libvirt patch already merged and without this patch it will raise error. Thanks

File: virttest/utils_misc.py [hunk 2]
```diff
@@ -4559,3 +4560,18 @@ def get_pid(name, session=None):
...
+
+def start_rsyslogd():
+    """
+    Start rsyslogd service
+    """
+    try:
+        utils_path.find_command("rsyslogd")
+    except utils_path.CmdNotFoundError:
+        exceptions.TestError("No rsyslogd command found.")
+    rsyslogd = service.Factory.create_service('rsyslog')
```
Comment (chunfuwen) [inline virttest/utils_misc.py]: Do you need first check whether one rsyslogd service is already running before you create one?

Comment (balamuruhans) [reply virttest/utils_misc.py]: the check is taken care in the next line.

**Pull request merged**